src/constants.ts

```diff
@@ -105,13 +105,12 @@ export const TOKEN_INFO_VERSION = 1;
 /**
  * Max value (inclusive) before having to use 8 bytes: 2147483648 ~= 2.14748e+09
  */
-export const MAX_OUTPUT_VALUE_32: OutputValueType = 2 ** 31 - 1;
+export const MAX_OUTPUT_VALUE_32: OutputValueType = 2n ** 31n - 1n;
 
 /**
  * Max accepted value for an output
- * Because of a precision problem in javascript we don't handle all 8 bytes of value
  */
-export const MAX_OUTPUT_VALUE: OutputValueType = 2 ** 43;
+export const MAX_OUTPUT_VALUE: OutputValueType = 2n ** 63n;
```

Comment:
We do, because the first bit is used to signal the value size, so when using 8 bytes the maximum value is not `2**64-1`, but rather `2**63`. This is enforced in `hathor-core`.